packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -300,6 +316,22 @@ export function unpackRuntimeMessage(message: ISequencedDocumentMessage) {
     return message;
 }
 
+class LoadSequenceNumberMismatchError extends Error implements ICriticalContainerError, ILoggingError {
```

Comment:
Do we really need a separate error type? Can this be logged as a data corruption error instead?

Reply:
Good call, yes I changed it.